src/intervals/rounding.jl

```diff
@@ -2,74 +2,176 @@
 # E.g.  +(a, b, RoundDown)
 # Some, like sin(a, RoundDown)  are already defined in CRlibm
 
+# Rounding modes:
+# - :correct  # correct rounding
+# - :fast     # fast rounding by prevfloat and nextfloat  (slightly wider than needed)
+# - :none     # no rounding at all for speed
 
-# import Base: +, -, *, /, sin, sqrt, inv, ^, zero, convert, parse
 
+doc"""Choose rounding mode based on environment variable"""
 
-# unary minus:
--{T<:AbstractFloat}(a::T, ::RoundingMode) = -a  # ignore rounding
+function setup_rounding_mode()
+    if haskey(ENV, "VN_ROUNDING")
+        rounding_mode = Symbol(ENV["VN_ROUNDING"])
```

Comment:
This function should be called only when the package is being loaded. The only (current) reason for making it into a function is for tidiness. I am hoping that at some point it will be possible to turn it into a function that may be used interactively, but according to the discussions I have had, it doesn't seem possible currently.